### ISSUE TITLE
Issue in Broadcast example

### ISSUE THREAD

**Issue by FelixNicolaeBucsa** [fetchai/uAgents#433]
### Prerequisites

- [X] I checked the [documentation](https://fetch.ai/docs) and found no answer to my problem
- [X] I checked the [existing issues](https://github.com/fetchai/uAgents/issues) and made sure there are no similar bug reports

### Category

Bug (unexpected behavior)

### Observed Behavior

In output i get the following:

```
INFO:     [alice]: Almanac registration is up to date!
INFO:     [  bob]: Almanac registration is up to date!
INFO:     [charles]: Almanac registration is up to date!
ERROR:    [charles]: No active agents found for: proto:5ec877056f701fdbae45bc2fe702adb58a502c81a7ba8c0d65c58998725544e5
INFO:     [charles]: Trying to contact 0 agents.
INFO:     [bureau]: Starting server on http://0.0.0.0:8000 (Press CTRL+C to quit)
ERROR:    [charles]: No active agents found for: proto:5ec877056f701fdbae45bc2fe702adb58a502c81a7ba8c0d65c58998725544e5
INFO:     [charles]: Trying to contact 0 agents.
```

### To Reproduce

Run Broadcast.py example within `uagents` Github repo. Example i at: `uAgents/python/examples/14-broadcast`

### Version

0.11.1

**Comment by Archento** [comment]
Just tried the example and it works just fine.

```bash
INFO:     [alice]: Registering on almanac contract...
INFO:     [alice]: Registering on almanac contract...complete
INFO:     [  bob]: Registering on almanac contract...
INFO:     [  bob]: Registering on almanac contract...complete
INFO:     [charles]: Registering on almanac contract...
INFO:     [charles]: Registering on almanac contract...complete
INFO:     [bureau]: Starting server on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     [charles]: Received response from agent1qww3ju3h6kfcuqf54gkghvt2pqe8qp97a7nzm2vp8plfxflc0epzcjsv79t: Hello from alice
INFO:     [charles]: Received response from agent1q0mau8vkmg78xx0sh8cyl4tpl4ktx94pqp2e94cylu6haugt2hd7j9vequ7: Hello from bob
INFO:     [charles]: Trying to contact 2 agents.
INFO:     [charles]: Received response from agent1qww3ju3h6kfcuqf54gkghvt2pqe8qp97a7nzm2vp8plfxflc0epzcjsv79t: Hello from alice
INFO:     [charles]: Received response from agent1q0mau8vkmg78xx0sh8cyl4tpl4ktx94pqp2e94cylu6haugt2hd7j9vequ7: Hello from bob
INFO:     [charles]: Trying to contact 2 agents.
...
```

Did you consider the note in https://github.com/fetchai/uAgents/blob/main/python/examples/14-broadcast/main.py#L31-L32 ?

**Comment by FelixNicolaeBucsa** [comment]
Yes. I have tried running that example straightforwardly but i am still getting the following after 5 minutes:

```
ERROR:    [charles]: No active agents found for: proto:5ec877056f701fdbae45bc2fe702adb58a502c81a7ba8c0d65c58998725544e5
INFO:     [charles]: Trying to contact 0 agents.
```

**Comment by FelixNicolaeBucsa** [comment]
<img width="944" alt="image" src="https://github.com/fetchai/uAgents/assets/72919584/9b04df98-9050-4294-9c98-568fec168853">

**Comment by Archento** [comment]
I've just tested this again with new agents and a new protocol.
<img width="1109" alt="image" src="https://github.com/fetchai/uAgents/assets/54712171/9b7586fa-ba4f-473d-b2a3-07d6a11e15b3">

Depending on the current load of the Almanac Smart Contract this may take more than 5 minutes but it will work eventually and as long as the protocol digest does not change it will work after restarting the agents.

The broadcast scenario covers the whole lifetime of an agent that will be discoverable by its supported protocols from which the first couple of minutes are less significant. This agent will typically provide a service or interface and is meant to run indefinitely.

Closing this issue as resolved.
